integration-tests/olminstall/helpers/runner.py

```
@@ -0,0 +1,785 @@
+            pod_obj = self.ka.get_json(f"/api/v1/namespaces/{quote(self.args.namespace)}/pods/{quote(pod)}")
+            containers = [c.get("name", "") for c in pod_obj.get("spec", {}).get("initContainers", [])] + [
+                c.get("name", "") for c in pod_obj.get("spec", {}).get("containers", [])
+            ]
+            for ctr in containers:
+                if ctr in {"prepare", "place-scripts", "place-tools"}:
+                    continue
+                print(f"\n[{task_name} : {ctr}]")
+                print(self.ka.get_text(f"/api/v1/namespaces/{quote(self.args.namespace)}/pods/{quote(pod)}/log?container={quote(ctr)}"), end="")
+
+    def stream_live_logs(self) -> None:
+        self.log_file = tempfile.mktemp(prefix="olminstall-run.")
```

Ruff (0.15.12)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)